pkg/ovs/util.go

```diff
@@ -23,3 +23,11 @@ func trimCommandOutput(raw []byte) string {
 	output := strings.TrimSpace(string(raw))
 	return strings.Trim(output, "\"")
 }
+
+func LogicalRouterPortName(lr, ls string) string {
+	return fmt.Sprintf("%s-%s", lr, ls)
+}
+
+func LogicalSwitchPortName(lr, ls string) string {
```

Comment:
Is this func used?

Reply:
I want to replace all the `fmt.Sprintf("%s-%s", ls, lr)`, `fmt.Sprintf("%s-%s", lr, ls)` to `LogicalSwitchPortName`, `LogicalRouterPortName`

https://github.com/kubeovn/kube-ovn/blob/337f6e05a6b12277eafd86689a7d6604a9cc44ae/pkg/ovs/ovn-nbctl-legacy.go#L414
https://github.com/kubeovn/kube-ovn/blob/337f6e05a6b12277eafd86689a7d6604a9cc44ae/pkg/ovs/ovn-nbctl-legacy.go#L761
https://github.com/kubeovn/kube-ovn/blob/337f6e05a6b12277eafd86689a7d6604a9cc44ae/pkg/ovs/ovn-nbctl-legacy.go#L762
https://github.com/kubeovn/kube-ovn/blob/337f6e05a6b12277eafd86689a7d6604a9cc44ae/pkg/ovs/ovn-nbctl-legacy.go#L774
https://github.com/kubeovn/kube-ovn/blob/337f6e05a6b12277eafd86689a7d6604a9cc44ae/pkg/ovs/ovn-nbctl-legacy.go#L775
https://github.com/kubeovn/kube-ovn/blob/337f6e05a6b12277eafd86689a7d6604a9cc44ae/pkg/ovs/ovn-nbctl-legacy.go#L2506

Reply:
OK. The function name may not be so sutiable.